mathesar_ui/src/systems/table-view/row/CellErrors.svelte

```diff
@@ -61,13 +61,13 @@
   }
 
   path {
-    fill: rgba(255, 0, 0, 0.5);
+    fill: var(--red-800);
     stroke: none;
   }
 
   .errors {
-    background: #fef1f1;
-    border: solid 0.1em #ffa1a1;
+    background: var(--slate-100);
```

Comment:
These error messages look a little awkward with a slate background.

![image](https://user-images.githubusercontent.com/42411/210443685-0ea0a28b-1019-4f8a-84c3-f16625eb0247.png)

I think it looked better before:

![image](https://user-images.githubusercontent.com/42411/210443836-e7dd6050-792e-43fa-8505-cb455016f654.png)

Reply:
I agree it looked better before.

Reply:
I agree. This was a mistake when replacing color variables.

Reply:
I see you fixed this in 09ce8767a5a6f708044eae0596e678e4d5ec0097 so I'm marking this thread as resolved.